### ISSUE TITLE
projectile aimbot bug/issue

### ISSUE THREAD

**Issue by phonkv2** [rei-kes/Fedoraware#8]
So basically when im shooting using a projectile it does a funny and shoots the ground instead is this a config issue or a issue with the cheat not sure but yeah

**Comment by SkididSkider** [comment]
prob the cheat. I tried many type of cfgs and some times it does the same

**Comment by Ragdoll117** [comment]
I don't have this problem, Don't fully rely on the projectile aimbot predicting the target's next position, Time it right or otherwise it will predict the wrong location if you're just holding the aimbot key

**Comment by phonkv2** [comment]
i understand what youre saying but i do not use the aimbot to predict splash damage or the aimbot but it still aims down at the ground PS: i have no anti aim or anything on

**Comment by Ragdoll117** [comment]
do you have it set to feet?

**Comment by phonkv2** [comment]
like the aimbot set to aim at the feet?

**Comment by karawaya** [comment]
yeah, it seems to happen with me aswell. the only thing you can do until rei fixes its is to not jump while firing your rocket.

**Comment by karawaya** [comment]
> do you have it set to feet?

 i dont think you can set proj aimbot to feet thats not an  option  i believe.

**Comment by rei-kes** [comment]
> So basically when im shooting using a projectile it does a funny and shoots the ground instead is this a config issue or a issue with the cheat not sure but yeah

as in shoots directly below you or under the target?

**Comment by karawaya** [comment]
> > So basically when im shooting using a projectile it does a funny and shoots the ground instead is this a config issue or a issue with the cheat not sure but yeah
> 
> as in shoots directly below you or under the target?

Directly below you.

**Comment by SkididSkider** [comment]
> > > So basically when im shooting using a projectile it does a funny and shoots the ground instead is this a config issue or a issue with the cheat not sure but yeah
> > 
> > 
> > as in shoots directly below you or under the target?
> 
> Directly below you.

try to disable "Fast Strafe", this fixed it for me. Bassacly when you move(rare times) on when you jump it jitters your character. So there is a 50% chance of shooting in the direction you want to and a 50% chance of shooting below you.

**Comment by phonkv2** [comment]
> > So basically when im shooting using a projectile it does a funny and shoots the ground instead is this a config issue or a issue with the cheat not sure but yeah
> 
> as in shoots directly below you or under the target?

yes directly below you and for the "Fast strafe" i do not have that enabled thought it only happens a rare amount of times but when it does happen restarting reinjecting is really annoying

**Comment by rei-kes** [comment]
can you provide a config

**Comment by phonkv2** [comment]
sure here 
https://mega.nz/file/wMVlwR5R#cvilkAn9__afhafAiHxX0rK0YM6d9LYL-uWt5HL1vQk
though i will say its a rare occurence so yeah

**Comment by rei-kes** [comment]
i haven't been able to reproduce this, if you can could you send a video or specific situations this happens in

**Comment by phonkv2** [comment]
> i haven't been able to reproduce this, if you can could you send a video or specific situations this happens in

If i ever come across this again i will be sure to clip it though as of now i have no video for a situation for this to happen in im not sure it happens rather randomly

**Comment by phonkv2** [comment]
> i haven't been able to reproduce this, if you can could you send a video or specific situations this happens in

Hello i was able to somewhat reproduce this bug and got it on video as well 
[https://small.fileditchstuff.me/s6/uFSAeCFaAWZVgUvMWHT.mp4](https://embeds.video/https://small.fileditchstuff.me/s6/uFSAeCFaAWZVgUvMWHT.mp4)

**Comment by rei-kes** [comment]
> Hello i was able to somewhat reproduce this bug and got it on video as well [https://small.fileditchstuff.me/s6/uFSAeCFaAWZVgUvMWHT.mp4]

by the looks of it, this may be an antiaim issue. i'll look into this when i can.

**Comment by phonkv2** [comment]
> > Hello i was able to somewhat reproduce this bug and got it on video as well [https://small.fileditchstuff.me/s6/uFSAeCFaAWZVgUvMWHT.mp4]
> 
> 
> 
> by the looks of it, this may be an antiaim issue. i'll look into this when i can. 

okay thank you
